### ISSUE TITLE
Fontawesome/Dropdown/Accordion error

### ISSUE THREAD

**Issue by extremelygooddeveloper** [fomantic/Fomantic-UI#1254]
Getting two errors on firefox 72, does not appear to have an affect on the framework, but i haven't tested that much.
Happens on a fresh install of fomantic and only on firefox afaik.
```
downloadable font: OS/2: Bad sTypoLineGap, setting it to 0: -32 (font-family: "Dropdown" style:normal weight:400 stretch:100 src index:0) source: data:application/x-font-ttf;charset=utf-8;base64,AAEAAAALAIAAAwAwT1MvMggjB5AAAAC8AAAAYGNtYXAPfuIIAAABHAAAAExnYXNwAAAAEAAAAWgAAAA … AG0AbwBvAG4ARgBvAG4AdAAgAGcAZQBuAGUAcgBhAHQAZQBkACAAYgB5ACAASQBjAG8ATQBvAG8AbgAuAAAAAAMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=
downloadable font: OS/2: Bad sTypoLineGap, setting it to 0: -32 (font-family: "Accordion" style:normal weight:400 stretch:100 src index:0) source: data:application/x-font-ttf;charset=utf-8;base64,AAEAAAALAIAAAwAwT1MvMggjB5AAAAC8AAAAYGNtYXAPfOIKAAABHAAAAExnYXNwAAAAEAAAAWgAAAA … AHIAYQB0AGkAbgBnAEYAbwBuAHQAIABnAGUAbgBlAHIAYQB0AGUAZAAgAGIAeQAgAEkAYwBvAE0AbwBvAG4ALgADAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
```

**Comment by dutrieux** [comment]
This is an old problem : https://github.com/Semantic-Org/Semantic-UI/issues/2146

**Comment by sofiarosati** [comment]
Still happening, same exact message on firefox 73.

**Comment by MuddyFire** [comment]
I ran into this same problem in my nav menu. It, of course, has dropdowns to show additional submenus/options... and I'm using the Fomantic-UI `$(".ui.dropdown").dropdown();` as well...

I resolved the issue by changing all of my `<i class="dropdown icon">` tags to `<i class="caret down icon">`

...the keyword `dropdown`, in the context of an icon, is nonetheless being interpreted as a dropdown **menu**. You could also fix this by being more specific in the jQuery call...

**Comment by lubber-de** [comment]
Fixed by https://github.com/fomantic/Fomantic-UI/pull/2480

**Comment by lubber-de** [comment]
Still have to investigate again, just using woff files do unfortunately seem to not fix this firefox warning 😢

**Comment by lubber-de** [comment]
@mvorisek helped to finally fix this in #2480 👏🏼
